src/Router/Boot.tsx

```diff
@@ -0,0 +1,39 @@
+import { Theme, themeProps } from "@artsy/palette"
+import React from "react"
+import { GridThemeProvider } from "styled-bootstrap-grid"
+import { GlobalStyles } from "Styleguide/Elements/GlobalStyles"
+import { Grid } from "Styleguide/Elements/Grid"
+import { ResponsiveProvider } from "Styleguide/Utils/Responsive"
+import { Provider as StateProvider } from "unstated"
+import { ContextProvider } from "../Components/Artsy"
+import { GlobalState } from "./state"
+import { BootProps } from "./types"
+
+export const Boot: React.SFC<BootProps> = ({ children, ...props }) => {
+  const globalState = new GlobalState(props)
+
+  return (
+    <StateProvider inject={[globalState]}>
```

Comment:
Props passed in are injected into the state provider. This also includes various configuration items like routes, initialRoute, `relayEnvironment` etc, which is accessible under the `state.system` key if you subscribe to `GlobalState`.